modules/tile-converter/docs/README.md

```diff
@@ -44,10 +44,17 @@ Note: the command line tools are implemented using this API and offer the same f
 - [OGC I3S Indexed Scene Layer Standard](http://www.ogc.org/standards/i3s) - The official standard from [OGC](https://www.opengeospatial.org/), the Open Geospatial Consortium.
 - [OGC 3D Tiles Standard](https://www.opengeospatial.org/standards/3DTiles) - The official standard from [OGC](https://www.opengeospatial.org/), the Open Geospatial Consortium.
 
+## Build instructions
+
+There are two ways to perform conversion:
```

Comment:
It seems to me that we can cut this section now that we have a separate page. If not, we should perhaps explain why and when a user would want to build things themselves?